### DISCUSSION TITLE
Recipe sharing to another cookbook user

### DISCUSSION THREAD

**Discussion by Balinor23** [nextcloud/cookbook#1155] [General]
Hi, I recently started using cookbook and have most of my recipes there now.  My oldest son started using it too and I would like to share some of my recipes with him without re-entering them.  I couldn't find a way to do this.  Is this currently possible or something you may enable in the future?

Thanks

**Comment by seyfeb** [comment]
At the moment you may share recipes with other users on the same Nextcloud instance by sharing the Recipe folder as described [in the documentation](https://nextcloud.github.io/cookbook/user/) at the bottom. More (enhanced) sharing options might become available at some point in the future. For more information you might want to check #120!

**Reply by Balinor23** [reply]
Thanks for the quick reply.  I was hoping there was a way to share specific recipes but I'll setup the shared folder method.

**Reply by seyfeb** [reply]
Unfortunately, at the moment, there is no way to selectively sync single recipes with some selected subset of users.

**Reply by Benedikt81** [reply]
Why not putting a GUID into every recipe? When a link is created the GUID of the link target is also stored and used as a fallback if the recipe isn't found.

**Reply by christianlupus** [reply]
This is the idea in #340 to put a fixed value into each recipe.

**Reply by Joe-ncB** [reply]
I would like to have this feature too.
